cs3api4lab/api/locks/lock_api_factory.py

```diff
@@ -0,0 +1,14 @@
+from cs3api4lab.api.locks.local_lock_api import LocalLockApi
+
+
+class LockApiFactory:
+
+    @staticmethod
+    def create(log, config):
+        if config.locks_impl == 'local':
+            return LocalLockApi(log, config)
+        # todo implement
+        # elif config.locks_impl == 'cs3':
+        #     return Cs3LockApi()
+        else:

```

Comment:
Locks should always work, the basic version is working on metadata and if we choose we can go with grpc lock